server/src/Config.hs

```diff
@@ -66,6 +62,13 @@ configParser environment = Config
         value 256 <>
         help ("Smaller values decrease the risk of data loss during a crash, while " <>
               "higher values result in more requests being accepted in rapid succession."))
+  <*> optional (option timeDurationReader
+       (long "sync-interval" <>
+        metavar "DURATION" <>
+        help ("If supplied, data is only persisted to disc every DURATION time units." <>
+              "The units 'm' (minutes), 's' (seconds), 'ms' (milliseconds) " <>
+              "and 'mus' (microseconds) can be used. " <>
```

Comment:
I don't think microsecond intervals are useful.

Reply:
Yes, I would drop the microseconds here.

Reply:
Probably not, I can remove it if desired. But it doesn't hurt I think, because one always has to specify a unit. It cannot happen that someone accidentally syncs every 5 microseconds.